Gemfile

```diff
@@ -1,10 +1,3 @@
 source 'http://rubygems.org'
 
 gemspec
-
-case version = ENV['GRAPE_VERSION'] || '~> 0.14.0'
-when 'HEAD'
-  gem 'grape', github: 'ruby-grape/grape'
-else
-  gem 'grape', version
-end
```

Comment:
yeap, I totally agree with you …
but we should support grape 0.14.0 for a while, as I said, support for > 0.16.1 is on the road,
but we should give all a while to test actual grape-swagger version and improve it